File: latihanA/Processor.cpp
#include <bits/stdc++.h>

using namespace std;

class Processor {
    private:
        string name;
        int price;
    
    public:
        Processor() {
            this->name = "";
            this->price = 0;
        }

        Processor(string name, int price) {
            this->name = name;
            this->price = price;
        }

        void setName(string name) {
            this->name = name;
        }

        string getName() {
            return this->name;
        }

        void setPrice(int price) {
            this->price = price;
        }

        int getPrice() {
            return this->price;
        }

        void printProcessor() {
            cout << "Name\t\t: " << this->name << endl;
            cout << "Price\t\t: " << this->price << " IDR" << endl;
        }

        ~Processor() { }
};

File: latihanB/php/Processor.php
<?php 

class Processor {
    private $name,
            $price;
    
    function __construct($name = "", $price = 0) {
        $this->name = $name;
        $this->price = $price;
    }

    function setName($name) {
        $this->name = $name;
    }

    function getName() {
        return $this->name;
    }

    function setPrice($price) {
        $this->price = $price;
    }

    function getPrice() {
        return $this->price;
    }

    function printProcessor() {
        echo "Name\t\t: " . $this->name . "<br>";
        echo "Price\t\t: " . $this->price . " IDR<br>";
    }

    function __destruct() { }

}

?>